servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1071,6 +1104,17 @@ public String getUfsAddress() {
     return mWhitelist.getList();
   }
 
+  public List<Long> getLostFiles() {
+    Set<Long> lostFiles = Sets.newHashSet();
+    for (long blockId : mBlockMaster.getLostBlocks()) {
+      // the file id is the container id of the block id
+      long containerId = BlockId.getContainerId(blockId);
+      long fileId = InodeFile.toFileId(containerId);
```

Comment:
A high level suggestion: we should put all Id conversions probably in an `IdUtils` class, so it is easier to keep track their relationship rather than distributing them in different classes.